site/content/en/docs/tasks/operations/customize-envoyproxy.md

```diff
@@ -159,6 +159,71 @@ And also you can dynamically change the value.
 kubectl get deployment -l gateway.envoyproxy.io/owning-gateway-name=eg -n envoy-gateway-system
 ```
 
+## Customize EnvoyProxy deployment type
+
+By default, Envoy Gateway creates a EnvoyProxy Deployment, but it can also be configured to create a DaemonSet instead.
+
+To configure Envoy Gateway to deploy the EnvoyProxy as a DaemonSet, you need to specify a spec.provider.kubernetes.envoyDaemonset instead of a envoyDeployment.
```

Comment:
```suggestion
To configure Envoy Gateway to deploy the EnvoyProxy as a DaemonSet, you need to specify a `spec.provider.kubernetes.envoyDaemonset` instead of a `envoyDeployment`.
```